pkg/didcomm/common/service/service.go

```diff
@@ -83,6 +86,23 @@ func NewDIDCommMsg(payload []byte) (*DIDCommMsg, error) {
 	return msg, nil
 }
 
+// GetPthid fetches the parent threadID from the public did document for implicit invitations
+func GetPthid(didServices []did.Service) ([]did.Service, error) {
```

Comment:
function name is GetPthid (get parent thread ID), but returns did.Service. Looks like name is doesn't match the functionality.